cloudtrail/lib/botocore/data/iam/2010-05-08/examples-1.json

```diff
@@ -0,0 +1,1577 @@
+        "title": "To remove a tag from an IAM role"
+      }
+    ],
+    "UntagUser": [
+      {
+        "input": {
+          "TagKeys": [
+            "Dept"
+          ],
+          "UserName": "anika"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following example shows how to remove tags that are attached to a user named 'anika'.",
+        "id": "to-remove-a-tag-from-an-iam-user-1506719725554",
+        "title": "To remove a tag from an IAM user"
+      }
+    ],
+    "UpdateAccessKey": [
+      {
+        "input": {
+          "AccessKeyId": "AKIAIOSFODNN7EXAMPLE",
+          "Status": "Inactive",
+          "UserName": "Bob"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command deactivates the specified access key (access key ID and secret access key) for the IAM user named Bob.",
+        "id": "02b556fd-e673-49b7-ab6b-f2f9035967d0",
+        "title": "To activate or deactivate an access key for an IAM user"
+      }
+    ],
+    "UpdateAccountPasswordPolicy": [
+      {
+        "input": {
+          "MinimumPasswordLength": 8,
+          "RequireNumbers": true
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command sets the password policy to require a minimum length of eight characters and to require one or more numbers in the password:",
+        "id": "c263a1af-37dc-4423-8dba-9790284ef5e0",
+        "title": "To set or change the current account password policy"
+      }
+    ],
+    "UpdateAssumeRolePolicy": [
+      {
+        "input": {
+          "PolicyDocument": "{\"Version\":\"2012-10-17\",\"Statement\":[{\"Effect\":\"Allow\",\"Principal\":{\"Service\":[\"ec2.amazonaws.com\"]},\"Action\":[\"sts:AssumeRole\"]}]}",
+          "RoleName": "S3AccessForEC2Instances"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command updates the role trust policy for the role named Test-Role:",
+        "id": "c9150063-d953-4e99-9576-9685872006c6",
+        "title": "To update the trust policy for an IAM role"
+      }
+    ],
+    "UpdateGroup": [
+      {
+        "input": {
+          "GroupName": "Test",
+          "NewGroupName": "Test-1"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command changes the name of the IAM group Test to Test-1.",
+        "id": "f0cf1662-91ae-4278-a80e-7db54256ccba",
+        "title": "To rename an IAM group"
+      }
+    ],
+    "UpdateLoginProfile": [
+      {
+        "input": {
+          "Password": "SomeKindOfPassword123!@#",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- f3571dcf-ef83-4203-b6b6-dc09d3aff2c7 -->